compat/src/suspense.js

```diff
@@ -30,15 +30,15 @@ export function catchRender(error, newVNode, oldVNode) {
 	return false;
 }
 
-function removeDom(children) {
+function detachDom(children) {
```

Comment:
Renamed this to `detachDom` since it only detaches a subtree from the screen and doesn't remove every element